packages/teams-js/src/public/meeting.ts

```diff
@@ -94,29 +94,68 @@ export namespace meeting {
     type?: T;
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   * Data structure to represent call participant identifiers
+   */
+  interface ICallParticipantIdentifiers {
+    /**
+     * Phone number of a PSTN caller
+     */
+    phoneNumber?: string;
+
+    /**
+     * Email of a VoIP caller
+     */
+    email?: EmailAddress;
+  }
+
   /**
    * @hidden
    * Hide from docs
    * Data structure to represent call details
    */
   export interface ICallDetails extends IMeetingOrCallDetailsBase<CallType> {
     /**
+     * @deprecated please use {@link ICallDetails.originalCallerInfo} instead
+     *
      * @hidden
      * Phone number of a PSTN caller or email of a VoIP caller
      */
     originalCaller?: string;
 
     /**
+     * @hidden
+     * Object representing the original caller
+     */
+    originalCallerInfo?: ICallParticipantIdentifiers;
+
+    /**
+     * @deprecated please use {@link ICallDetails.dialedEntityInfo} instead
+     *
      * @hidden
      * Phone number of a PSTN callee or email of a VoIP callee
      */
     dialedEntity?: never;
 
+    /**
+     * @hidden
+     * Object representing the entity the caller dialed
+     */
+    dialedEntityInfo?: never;
```

Comment:
Can you explain why you had a parameter of type never and then added a new one also of type never? I don't think I understand your use of never here. Your PR description says:

"originalCaller represents the caller of a call, and dialedEntity represents the callee. Those two properties originally were of type string, and the return value could've been either an email or phone number in string format.

With this change, those properties have been updated to be of a new type ICallParticipantIdentifiers.
This new type ICallParticipantIdentifiers is an object that has 2 explicit fields for phoneNumber and email."

However, both dialedEntity and dialedEntityInfo are never of type `ICallParticipantIdentifiers`

Reply:
Those properties are in the process of being implemented, hence we originally decided to mark them as `never`, to indicate that they should not be used at this point in time. As the implementations are rolled out, we will gradually revert them back to their intended type.

Should we add that as context to the property description?

Regarding `dialedEntity`, it isn't being used now, but I was thinking I'd keep the "schema" consistent with `originalCaller` by adding the corresponding `dialedEntityInfo` property. That way, both the caller info and the callee (dialed entity) info would have the same naming convention.